src/axes/categoryAxis.ts

```diff
@@ -1,15 +1,7 @@
 namespace Plottable.Axes {
-  export interface DownsampleInfo {
-    domain: string[];
-    stepWidth: number;
-  }
+  export type ITickMarkPosition = "center" | "after";
```

Comment:
The rest of the discussion you linked settles on "between" though? My main concern with "after" is that it makes [this chart](https://cloud.githubusercontent.com/assets/1079495/22489635/998765fe-e7e6-11e6-9de9-790d939dd164.png) look bad